app/embeds/index.js

```diff
@@ -148,6 +149,13 @@ export default [
     component: GoogleSlides,
     matcher: matcher(GoogleSlides),
   },
+  {
+    title: "Giphy",
+    keywords: "short looping videos",
+    icon: () => <Img src="/images/giphy.png" />,
```

Comment:
I can't comment on the logo image itself due to GitHub limitations but it needs to be edge to edge of the 120px box otherwise it will look smaller than the rest